package.json

```diff
@@ -9,6 +9,7 @@
   },
   "files": [
     "index.js",
+    "index.d.ts",
```

Comment:
Also add it to a top-level "types" property next to the "main".

Reply:
Done